File: Tugas6/C++/Tugas6.2.cpp
//Nama		:Hilda Nurhidayati
//Npm		:20.14.1.0039
//Program	:Tugas6.2

#include<iostream>
using namespace std;

string nama, npm, jk, alamat, ttl, kontak_wa, email;
void biodata()
{
	cout<<"Masukkan Biodata"<<endl;
	cout<<"Nama					: ";
	getline (cin,nama);
	cout<<"Npm					: ";
	getline (cin,npm);
	cout<<"Jenis Kelamin (L/P) 	: ";
	getline (cin,jk);
	cout<<"Alamat				: ";
	getline (cin,alamat);
	cout<<"TTL					: ";
	getline(cin,alamat);
	cout<<"Kontak Whatsapp		: ";
	getline(cin,kontak_wa);
	cout<<"Email				: ";
	getline(cin,email);
	cout<<endl;
	
	cout<<"Nama Anda adalah				:"<<nama<<endl;
	cout<<"Npm Anda adalah				:"<<npm<<endl;
	cout<<"Jenis Kelamin Anda adalah	:"<<jk<<endl;
	cout<<"Alamat Anda adalah			:"<<alamat<<endl;
	cout<<"TTL Anda adalah				:"<<ttl<<endl;
	cout<<"Kontak Whatsapp Anda adalah	:"<<kontak_wa<<endl;
	cout<<"Email Anda adalah			:"<<email<<endl;
}
int main()
{
	biodata();
}

File: Tugas6/TugasFPC/tugas6.2.pas
{Nama           :Hilda Nurhidayati}
{NPM            :20.14.1.0039}
{Nama Program   :tugas6.2.pas}

program biodata2;
uses crt;

procedure biodata;
var
nama, npm, jk, alamat, ttl, kontak_wa, email:string;
begin
write('Masukkan Nama : ');readln(nama);
write('Masukkan Npm : ');readln(npm);
write('Masukkan Jenis Kelamin (L/P) : ');readln(jk);
write('Masukkan Alamat : ');readln(alamat);
write('Masukkan TTL : ');readln(ttl);
write('Masukkan Kontak WhatsApp : ');readln(kontak_wa);
write('Masukkan Email : ');readln(email);
writeln;
writeln('Biodata');
writeln('Nama                 : ',nama);
writeln('Npm                  : ',npm);
writeln('Jenis Kelamin        : ',jk);
writeln('Alamat               : ',alamat);
writeln('TTL                  : ',ttl);
writeln('Kontak Whatsapp      : ',kontak_wa);
writeln('Email                : ',email);
end;

begin
clrscr;
biodata;

readln;

end.